torchio/transforms/intensity_transform.py

```diff
@@ -1,11 +1,16 @@
+from typing import List, Dict
+
 from .transform import Transform
+from ..data import Image
 
 
 class IntensityTransform(Transform):
     """Transform that modifies voxel intensities only."""
-    @staticmethod
-    def get_images_dict(sample):
-        return sample.get_images_dict(intensity_only=True)
+    def get_images_dict(self, sample) -> Dict[str, Image]:
+        return sample.get_images_dict(intensity_only=True, include=self.include, exclude=self.exclude)
+
+    def get_images(self, sample) -> List[Image]:
```

Comment:
I don't think this method is necessary.

Reply:
@fepegar is there a reason that in spatial transforms (for example random_affine.py) this is used:

    ```
def apply_transform(self, subject: Subject) -> Subject:
        ...
        for image in self.get_images(subject):
```

and in intensity transforms (for example random_blur.py):
    ```
def apply_transform(self, subject: Subject) -> Subject:
        ...
        for name, image in self.get_images_dict(subject).items():
```
Could we make this consistent?

Reply:
The reason is that in spatial transforms you don't need to know the name of images because the same parameters are applied to all of them, whereas for intensity transforms you retrieve the name to store it in the parameters dict. Does that make sense?

Reply:
okay makes sense, will remove methods